tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -944,7 +951,8 @@ HRESULT CorProfiler::TryRejitModule(ModuleID module_id, std::vector<ModuleID>& m
             auto profiler_library_path = shared::GetEnvironmentValue(WStr("DD_INTERNAL_PROFILING_NATIVE_ENGINE_PATH"));
             if (!profiler_library_path.empty() && fs::exists(profiler_library_path))
             {
-                RewritingPInvokeMaps(module_metadata, WStr("continuous profiler"), profiler_nativemethods_type, profiler_library_path);
+                RewritingPInvokeMaps(module_metadata, WStr("continuous profiler"), profiler_nativemethods_type_in_profiler_native_module, profiler_library_path);
+                RewritingPInvokeMaps(module_metadata, WStr("continuous profiler"), profiler_nativemethods_type_in_tracer_native_module);
```

Comment:
Is it expected that "continuous profiler" is used here for tracer native module?

Reply:
Yep. The implementation of the new P/Invoke endpoint that I've added exist in the Tracer native module.